src/integrations/hapiApollo.ts

```diff
@@ -137,9 +152,75 @@ async function processQuery(body, optionsObject) {
       responses.push({ errors: [formatErrorFn(e)] });
     }
   }
-  return responses;
+  return reply(responses);
 }
 
 function isOptionsFunction(arg: ApolloOptions | HAPIOptionsFunction): arg is HAPIOptionsFunction {
   return typeof arg === 'function';
 }
+
+function createErr(code: number, message: string) {
+  const err = Boom.create(code);
+  err.output.payload.message = message;
```

Comment:
so I struggled with this for a while.  When I use Boom.create(500, 'Something went wrong') the payload returned simply says `Internal server error`.  I had to do this to change the message of the payload.  Thoughts?